docs/migrations/migrate_renderer.md

```diff
@@ -0,0 +1,228 @@
+---
+id: migrate_renderer
+title: From legacy to Decentralize renderer
+sidebar_label: Renderer
+---
+
+After the release of [Opencerts v2](/docs/migrations/v1_to_v2.md), institutions now require to host their own decentralized renderer. In this guide we will learn how to migrate a template from the legacy renderer that was hosted on [govtech repositories](https://github.com/OpenCerts/legacy-templates) into your own renderer. More specifically, we will migrate the [2018 Geekout](https://github.com/OpenCerts/legacy-templates/tree/master/src/components/CertificateTemplates/tlds/sg/gov/tech/2018-Geekout) template.
```

Comment:
"For example, we will be migrating the [2018 Geekout](https://github.com/OpenCerts/legacy-templates/tree/master/src/components/CertificateTemplates/tlds/sg/gov/tech/2018-Geekout) template."

Reply:
"GovTech repositories"